lib/entity_manager.js

```diff
@@ -22,6 +22,9 @@ function EntityManager(components, storage) {
 
   this._entityPool = []
   this._entityCounter = 0
+
+  this._entityCache = {}
+  this._entityCache[0] = []
```

Comment:
Can be optimized:

```js
this._entityCache = FastMap()
this._entityCache[0] = []
```

See https://gist.github.com/ooflorent/c58c9a0f08923973a5e3